estate/models/estate_property.py

```diff
@@ -0,0 +1,41 @@
+from odoo import fields, models
+
+
+'''
+name X Char
+description Text
+postcode Char
+date_availability Date
+expected_price Float
+selling_price Float
+bedrooms Integer 
+living_area Integer 
+facades Integer
+garage Boolean
+garden Boolean
+garden_area Integer
+arden_orientation Selection
+'''
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Estate property"
+
+    name = fields.Char(string='Estate Name', required=True)
+    description = fields.Text(string='Description')
+    postcode = fields.Char(string='Postcode')
+    date_available = fields.Date(string='Date Available')
+    expected_price = fields.Float(string='Expected Price', required=True)
+    selling_price = fields.Float(string='Selling Price')
+    bedrooms = fields.Integer(string='Bedrooms')
+    living_area = fields.Integer(string='Living Area')
+    facades = fields.Integer(string='Facades')
+    garage = fields.Boolean(string='Garage')
+    garden = fields.Boolean(string='Garden')
+    garden_area = fields.Integer(string='Garden Area')
+    arden_orientation = fields.Selection(string='Arden Orientation', selection=[
+        ('north', 'North'),
+        ('south', 'South'),
+        ('east', 'East'),
+        ('west', 'West')
```

Comment:
In python, we try to follow the following convention for quoting strings:
- single quote for technical terms
- double quotes for displayed texts

=> `('north', "North")`